modules/installation-azure-stack-hub-config-yaml.adoc

```diff
@@ -113,13 +123,22 @@ baseDomain: example.com <1>
 credentialsMode: Manual
 controlPlane: <2> <3>
   name: master
+  platform:
+    azure:
+      osDisk:
+        diskSizeGB: 1024 <4>
+        diskType: premium_LRS
   replicas: 3
 compute: <2>
 - name: worker
-  platform: {}
+  platform:
+    azure:
+      osDisk:
+        diskSizeGB: 512 <4>
+        diskType: premium_LRS
   replicas: 3
 metadata:
-  name: test-cluster <1> <4>
+  name: test-cluster <1> <5>
```

Comment:
I can't find a rule on this, but I brought it up in our PR channel. I don't think, and neither did someone else, that we should be doing two callouts per parameter. I think that would be be better suited as something like:

region: azure_stack_local_region <8>
----
<8>. Required. The name of your Azure Stack Hub local region.

That said, there are no requirements to call out whether something is required, so you could omit it completely. Alternatively, you could do the 

alternate callout  method:

----
code <variable_1>
code <variable_2>
----
+
where:

<variable_1>:: Specifies the explanation of the first variable.
<variable_2>:: Specifies the explanation of the first variable. 

Found here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output